runtime/jvmti/jvmtiStartup.c

```diff
@@ -168,7 +168,7 @@ createAgentLibraryWithOption(J9JavaVM *vm, J9VMInitArgs *argsList, IDATA agentIn
 #if defined(J9VM_OPT_CRIU_SUPPORT)
 		if ((JNI_OK == result)
 			&& (isXrunjdwp || (0 == strncmp(JDWP_AGENT, optionsPtr, libraryLength)))
-			&& (vm->internalVMFunctions->isDebugOnRestoreEnabled(vm->mainThread))
+			&& (vm->internalVMFunctions->isDebugOnRestoreEnabled(vm))
 		){
```

Comment:
While you're here, please add the missing space before `{`.

Reply:
Added